Makefile.in

```diff
@@ -3,7 +3,9 @@ VERSION=$(shell ./genver.sh -r)
 
 # Configuration -- you probably need to `make clean` if you
 # change any of these
-# landlock is checked by `configure` and set in config.h
+
+# uncomment the following line to disable landlock
+# override undefine HAVE_LANDLOCK
```

Comment:
Ideally I'd prefer to include options to `configure` so we have `./configure --disable-landlock`, but after a quick look it seems not as easy  as it should be... I'll look into it later.